tests/plugins/keyword_test.py

```diff
@@ -64,7 +64,7 @@ def test_analyze_positives(self, file_content):
         'file_content',
         [
             (
-                'private_key \'something\';'  # Single-quotes not double-quotes
+                'private_key \'"no spaces\';'  # Has whitespace in-between
```

Comment:
Is the `"` expected?

Reply:
@domanchi Do we want to have a regex for single-quotes, and a regex for double-quotes? I can't think of either way I'm happy with.
e.g. `private_key_for_blah "grammar' messes thing's up bad"` with this regex only returns `grammar`, whereas if we had 2 there would be 2 matches. One for the whole string (the double-quotes, which is good), and one for the single-quotes (which is ` messes thing`, and bad).
Unless we could somehow have a stack, and only match, the 2nd time, the first type of quote that was matched. I'll see how hard that is.

We have only seen it followed by double-quotes though, so maybe we should stick to that?